tests: drivers: counter_basic_api: Build failing on LPCxpresso55s69_cpu

**Describe the bug**
counter_basic_api test fails to build targeting LPCxpresso55s69 EVK.

**To Reproduce**
Steps to reproduce the behavior:
1. mkdir build; cd build
2. cmake -DBOARD=lpcxpresso55s69_cpu0 tests/drivers/counter/counter_basic_api
3. make
4. See error

**Expected behavior**
The build and test should pass.

**Logs and console output**
Log from the build failure:
```
[113/179] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_syscon.c.obj
FAILED: zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_syscon.c.obj
ccache /opt/zephyr-sdk-0.13.2/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DBOARD_FLASH_SIZE="CONFIG_FLASH_SIZE*1024" -DCPU_LPC55S69JBD100_cm33_core0 -DKERNEL -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR_SUPERVISOR__ -D__ZEPHYR__=1 -I/home/danieldegrasse/zephyrproject/zephyr/include -I/home/danieldegrasse/zephyrproject/zephyr/build/zephyr/include/generated -I/home/danieldegrasse/zephyrproject/zephyr/soc/arm/nxp_lpc/lpc55xxx -I/home/danieldegrasse/zephyrproject/zephyr/subsys/testsuite/include -I/home/danieldegrasse/zephyrproject/zephyr/subsys/testsuite/ztest/include -I/home/danieldegrasse/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/LPC55S69 -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/devices/LPC55S69/drivers -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/common -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/ctimer -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/lpc_gpio -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/pint -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/inputmux -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/flexcomm -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/rng_1 -I/home/danieldegrasse/zephyrproject/modules/hal/nxp/mcux/mcux-sdk/drivers/lpc_iocon -isystem /home/danieldegrasse/zephyrproject/zephyr/lib/libc/minimal/include -isystem /opt/zephyr-sdk-0.13.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/include -isystem /opt/zephyr-sdk-0.13.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/10.3.0/include-fixed -Os -imacros /home/danieldegrasse/zephyrproject/zephyr/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m33 -mthumb -mabi=aapcs -mfp16-format=ieee -imacros /home/danieldegrasse/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/home/danieldegrasse/zephyrproject/zephyr/tests/drivers/counter/counter_basic_api=CMAKE_SOURCE_DIR -fmacro-prefix-map=/home/danieldegrasse/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/home/danieldegrasse/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -mcmse -std=c99 -nostdinc -MD -MT zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_syscon.c.obj -MF zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_syscon.c.obj.d -o zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_control_mcux_syscon.c.obj -c /home/danieldegrasse/zephyrproject/zephyr/drivers/clock_control/clock_control_mcux_syscon.c
/home/danieldegrasse/zephyrproject/zephyr/drivers/clock_control/clock_control_mcux_syscon.c: In function 'mcux_lpc_syscon_clock_control_get_subsys_rate':
/home/danieldegrasse/zephyrproject/zephyr/drivers/clock_control/clock_control_mcux_syscon.c:137:2: error: duplicate case value
  137 |  case MCUX_BUS_CLK:
      |  ^~~~
/home/danieldegrasse/zephyrproject/zephyr/drivers/clock_control/clock_control_mcux_syscon.c:126:2: note: previously used here
  126 |  case (MCUX_CTIMER2_CLK + MCUX_CTIMER_CLK_OFFSET):
      |  ^~~~
[122/179] Linking C static library zephyr/drivers/timer/libdrivers__timer.a
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/danieldegrasse/zephyrproject/zephyr/build
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04 (WSLv1)
 - Toolchain: Zephyr SDK 0.13.2
 - Commit e20f72f7cba428d3d75ddc86ff946e24fb7ccc27

--- a/include/dt-bindings/clock/mcux_lpc_syscon_clock.h
+++ b/include/dt-bindings/clock/mcux_lpc_syscon_clock.h
@@ -35,8 +35,8 @@
 #define MCUX_CTIMER3_CLK		3
 #define MCUX_CTIMER4_CLK		4
 
-#define MCUX_MCAN_CLK			23
+#define MCUX_MCAN_CLK			27
 
-#define MCUX_BUS_CLK			24
+#define MCUX_BUS_CLK			28
 
 #endif /* ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_MCUX_LPC_SYSCON_H_ */

